imageUploadToCloud.js

```diff
@@ -0,0 +1,94 @@
+const { Storage } = require("@google-cloud/storage");
```

Comment:
@aman-tunerlabs - Why not use Kendra service for the uploads, why create a new file ?

Reply:
@aks30 Earlier Comment says to create a seperate file for image upload to cloud. But latest comment says to use kendra service for upload.